Python/setup.py

```diff
@@ -41,29 +41,36 @@
 import os
 import setuptools
 
+
+def read_requirements():
+    requirements = []
+    with open('./requirements.txt', 'r') as file:
+        for line in file:
+            line = str(line)  # guard against str exceptions
+            line = line[:line.find("#")] if "#" in line else line  # exclude comments
+            line = line.strip()  # clean
+            if line:
```

Comment:
One line if it fits in 80 characters please

Reply:
Shortened all 80+ char lines. ...